tests/subsys/tracing/tracing_sys/testcase.yaml

```diff
@@ -0,0 +1,8 @@
+tests:
+  tracing.disable:
+     extra_args: CONF_FILE="prj.conf"
```

Comment:
Why do we need this?

Reply:
Hi @stephanosio , thanks for your comments, I have corrected spelling errors. Here is to test the mutex/semaphore behavior when tracing is disabled, and then enable tracing, and verify that tracing does not interfere with the system behavior by comparison.